### ISSUE TITLE
Difference between accuracy and categorical_accuracy

### ISSUE THREAD

**Issue by jan-christiansen** [sagr4019/ResearchProject#57]
The implementation of "Finding and correcting syntax errors using recurrent neural networks" uses categorical_accuracy while the implementation of "Sequence Classification with LSTM" uses accuracy. How do these two work and what is the difference.

**Comment by Salvos** [comment]
Added a wiki article for **all** keras metrics 
https://github.com/sagr4019/ResearchProject/wiki/Keras-accuracy-(metrics)

[Sergii Gryshkevych from StackOverflow](https://stackoverflow.com/a/41534323) refered that the default type "accuracy" is determined in the **training.py** and the default choice is the **categorial_accuracy**.

**Comment by jan-christiansen** [comment]
Please move a short (a couple of sentences) explanation to [General Terminology](../wiki/general-terminology).

**Comment by Salvos** [comment]
Added the explanation as a subtitle of "Accuracy": 
https://github.com/sagr4019/ResearchProject/wiki/General-Terminology#difference-between-accuracy-and-categorical_accuracy

**Comment by jan-christiansen** [comment]
This issues referred to the output of the two implementations as seen [here](https://github.com/sagr4019/ResearchProject/issues/48) and [here](https://github.com/sagr4019/ResearchProject/issues/52). In other words, what do the numbers, reported by the two implementation, mean. Your text still does not explain this.

In the sentence "This decision is based on certain parameters like the output shape and the loss functions." what is the "output shape"? The sentence "The metric categorical_accuracy is a function to judge the performance of the model on a given algorithm." is ok but does not explain how the judgement works.

**Comment by Salvos** [comment]
Updated the subtitle [Difference between accuracy and categorical_accuracy](https://github.com/sagr4019/ResearchProject/wiki/General-Terminology#difference-between-accuracy-and-categorical_accuracy)

- added literal description for "categorical accuracy"
- added literal description for "output shape"
